docs/man-openmpi/man3/MPI_Info_set.3.rst

```diff
@@ -20,7 +20,7 @@ C Syntax
 
    #include <mpi.h>
 
-   int MPI_Info_set(MPI_Info info, char *key, char *value)
+   int MPI_Info_set(MPI_Info info, const char *key, const char *value)
```

Comment:
Out side the scope of this pr - why are some of these in italics and some of them are not? Seems random.